Tasks/Galetskaya-Olga/Task4/Task4.py

```diff
@@ -0,0 +1,57 @@
+while True:
+    user_value = input('Введите максимальное количество символов в строке (больше 35):\n')
+
+    failed = False
+    if  not user_value.isdigit():
+        failed = True
+        if failed:
+            print('Вы ввели значение неверно, введите цифру больше 35 еще раз.\n')
+            continue
+    if int(user_value) <= 35:
+        print('Вы ввели значение неверно, введите цифру больше 35 еще раз.\n')
+        continue
+    break
+
+max_symbols = int(user_value)
+
+with open(r'Tasks\Galetskaya-Olga\Task4\text.txt', 'r', encoding='utf-8') as file:
```

Comment:
a bit overworked path to the file, not a mistake though, but it would be better to have it in a less specific way like 'text.txt'